docs/content/concepts/authentication/index.md

```diff
@@ -0,0 +1,73 @@
+---
+description: >
+  How to authenticate requests to kcp.
+---
+
+# Authentication
+
+KCP implements the same authentication mechanisms as Kubernetes, allowing the use of Kubernetes authentication strategies. The KCP server can also be configured to generate a local admin.kubeconfig file and a token hash file, enabling access to KCP as a shard admin. This authentication mechanism is then added to any existing Kubernetes authentication strategies from generic control plane settings.
```

Comment:
Minor nit: It's "kcp" in the description of the page and "KCP" in the body. Let's keep it consistent (I know our docs aren't good at this), so far we stylised as "kcp" (lower case).